web-registry/src/pages/Home.tsx

```diff
@@ -86,7 +86,7 @@ const Home: React.FC = () => {
       />
 
       {projectsData?.allProjects?.nodes && (
-        <Section title="Projects" titleAlign="left" classes={{ root: styles.section, title: styles.title }}>
+        <Section title="Featured Projects" classes={{ root: styles.section, title: styles.title }}>
```

Comment:
we display all projects by default right now (just fetching all existing projects in the pg database) so if we want to only display some featured projects in the future, we'll need an issue to do that, it could be configurable through Sanity for instance cc/ @erikalogie (although maybe that's not so urgent given we don't have a ton of projects yet)